tests/_data/models.py

```diff
@@ -737,6 +748,64 @@ def get_component_setuptools_complete(include_pedigree: bool = True) -> Componen
     return component
 
 
+def get_component_with_evidence(include_pedigree: bool = True) -> Component:
+    component = get_component_setuptools_simple()
+    component.evidence = get_component_evidence_basic()
+    return component
+
+
+def get_component_evidence_basic() -> ComponentEvidence:
+    """
+    Returns a basic ComponentEvidence object for testing.
+    """
+    return ComponentEvidence(
+        identity=[
+            Identity(
+                field=IdentityFieldType.NAME,
+                confidence=Decimal('0.9'),
+                concluded_value='example-component',
+                methods=[
+                    Method(
+                        technique=AnalysisTechnique.SOURCE_CODE_ANALYSIS,
+                        confidence=Decimal('0.8'), value='analysis-tool'
+                    )
+                ],
+                tools=[
+                    BomRef('ref0'),  # BomRef reference
```

Comment:
this is not how this should work.
instead of rafting a new `BomRef` for this single use,
the spec clearly says to reference an already defined one that was used in `metadata.tools`.
https://cyclonedx.org/docs/1.6/json/#components_items_evidence_identity_oneOf_i0_items_tools
> The object in the BOM identified by its bom-ref. This is often a component or service but may be any object type supporting bom-refs. Tools used for analysis should already be defined in the BOM, either in the metadata/tools, components, or formulation.